backend/core/src/applications/services/geocoding.service.spec.ts

```diff
@@ -175,7 +176,7 @@ describe("GeocodingService", () => {
           multiselectQuestion: {
             options: [
               {
-                text: "Geocoding option by radius",
+                text: "Geocoding option by Radius",
```

Comment:
Nit: Any reason for the case change here? See there is still a lowercase "radius" and all the "map"s are lowercased

Reply:
This was changed because I misread an error when testing these tests. I thought I had mistyped 'radius' so I retyped it but accidentally capitalized it. The error was unrelated to that and I just never switched it back